hathor/cli/events_simulator/scenario.py

```diff
@@ -14,63 +14,119 @@
 
 from enum import Enum
 
-from hathor.event.model.base_event import BaseEvent
-from hathor.event.model.event_data import TxData, TxMetadata
-from hathor.event.model.event_type import EventType
+from hathor.event.model.base_event import BaseEvent, EventType
+from hathor.event.model.event_data import EmptyData, ReorgData, SpentOutput, TxData, TxInput, TxMetadata, TxOutput
 
+_ONLY_LOAD_EVENTS = [
```

Comment:
Should we move it to another file with some notes on how to generate it again?

Reply:
Sure, done in https://github.com/HathorNetwork/hathor-core/pull/609/commits/46f66c7e961e8fc2d200626d7fa51bc3f64c3ba1

Reply:
Thanks!